tests/unit/unit-util.cpp

```diff
@@ -156,6 +156,31 @@ TEST_P(UnitUtilIntTests, Oracles)
   }
 }
 
+TEST_P(UnitUtilTests, cnf_to_dimacs)
```

Comment:
This is a great test.
Let's add some more, with some corner cases, for example:
- a formula with only one clause
- a formula with no clauses
- a clause with only one literal
- an empty clause